tests/005-test-devmapper-cleanup

```diff
@@ -0,0 +1,61 @@
+source $SRCDIR/libtest.sh
+
+cleanup_local() {
+  local test_status
+  local vg_name=$1
+  local devs=$2
+  $DSSBIN --reset >> $LOGS 2>&1
+  test_status=0
+  if [ -e /etc/sysconfig/docker-storage ]; then
+     echo "  $DSSBIN reset Failed: $(basename testname)"
+     test_status=1
+  fi
+  cleanup $vg_name $devs
+  echo $test_status
+}
+
+# Test DEVS= directive. Returns 0 on success and 1 on failure.
+test_devs() {
+  local devs=$DEVS
+  local test_status
+  local testname=`basename "$0"`
+  local vg_name
+  local vg_name="dss-test-dev"
+
```

Comment:
I got a failure on this test, which is why I changed it, I will change it back and see if tests succeed.